### ISSUE TITLE
Download all users in a list

### ISSUE THREAD

**Issue by AplhaSlayer1964** [AssetKid/raider-release#71]
Would it be possible to add the ability to have have Raider take in a list of users to download everything from? I'm looking to switch to Raider from instaloader but it's going to take a bit to manually add every account even if it is just one time. If there is already a way to do this I couldn't find it.

**Comment by quietsecret123** [comment]
It's probably not the automated way you wanted, but this might be a start: https://github.com/AssetKid/raider-release/issues/3#issuecomment-526864700

I find that typing it in the acc file is far less laggy and less time consuming than typing them into Raider one by one.

But there's no automated way to import from a list, to my knowledge.

**Comment by AplhaSlayer1964** [comment]
Thanks I will try doing that. Should be a lot faster especilly with how many accounts I'm trying to download.

**Comment by quietsecret123** [comment]
> Thanks I will try doing that. Should be a lot faster especilly with how many accounts I'm trying to download.

If you already have the user IDs, it's even faster. Type them as

`{"name":"foldername","username":"username","userid":"numbers","updated":1}`

e.g. 

`{"name":"chrishemsworth","username":"chrishemsworth","userid":"1822594644","updated":1}`

Since they will all be last updated 51 years ago (ie Unix timestamp of 1), Raider will download them in the order in which they are typed. This will save you the step of clicking "save" on each folder.

**Comment by AplhaSlayer1964** [comment]
> > Thanks I will try doing that. Should be a lot faster especilly with how many accounts I'm trying to download.
> 
> If you already have the user IDs, it's even faster. Type them as
> 
> `{"name":"foldername","username":"username","userid":"numbers","updated":1}`
> 
> e.g.
> 
> `{"name":"chrishemsworth","username":"chrishemsworth","userid":"1822594644","updated":1}`
> 
> Since they will all be last updated 51 years ago (ie Unix timestamp of 1), Raider will download them in the order in which they are typed. This will save you the step of clicking "save" on each folder because Raider will prompt you to know what you want each folder (for each username) to be named as.

I managed to just manually add all of them (took a really long time) but I do have a question. When I tell it to update all the number for total is not the total posts downloaded but the total accounts being downloaded correct?

**Comment by quietsecret123** [comment]
Correct, it's total number of accounts in that acc file. You don't get to actually see how many posts per account is downloaded, but you do see your "lifetime" download count at the bottom (above "Donate Today").

**Comment by quietsecret123** [comment]
Off topic, but there's no way to DM you on here, it seems – If you don't mind me asking, what made you switch over from Instaloader?

**Comment by AplhaSlayer1964** [comment]
> Off topic, but there's no way to DM you on here, it seems – If you don't mind me asking, what made you switch over from Instaloader?

Sorry since the thread got closed I didn’t see you responded on my main page. I switched from instaloader because with the amount of accounts I'm saving (175ish)  I would get rate limited supper fast because instaloader gets a higher quality version when logged in (need to be logged in for some accounts). I also noticed even if I didn’t log in on instaloader I would hit some other api limit probably due to me not being logged in. I did prefer instaloader as I could set up a script to run when I want it to unlike the set times with raider. I will probably switch back when they add “lower” quality when logged in which they said they are working on adding.
